### DISCUSSION TITLE
Start new project with v5 or v6?

### DISCUSSION THREAD

**Discussion by c-gross** [fabricjs/fabric.js#9788] [General]
Hi, I wrote a complex product configurator many years ago when v1 was the current version. Also used v2 and v3 abit but not much.
I would like to rewrite the product configurator to implement many new features. But I'm not sure if v6 is near a final release since it's already 1 year in beta. 

Will a v5 to v6 upgrade be easy possible or is it a very big change like in a few earlier versions?

**Comment by asturur** [comment]
Use V6 since is done.
There will be more breaking changes but nothing so chaotic like from v5 to v6.

**Reply by c-gross** [reply]
Ok cool, sounds like it's stable and can be used in production. 
Is there an estimation for the final version?

**Reply by asturur** [reply]
We are arguing if to merge last one or two pr and we cut the line.
There is still a bunch to do on documentation.
There will be future breaking changes of course, but ONE per release and only after documentation has been renewed
